frontend-react/src/components/Table/Table.tsx

```diff
@@ -63,15 +56,22 @@ export interface ColumnConfig {
 
 export interface TableConfig {
     columns: Array<ColumnConfig>;
-    rows: Array<TableRow>;
+    rows: Array<StringIndexed>;
 }
 
 export interface DatasetAction {
     label: string;
     method: Function;
 }
 
-export type RowSideEffect = (row: TableRow | null) => Promise<void>;
+export interface ColumnProps {
```

Comment:
Crap. This was probably a merge conflict resolution issue. Guess I misread a change and this snuck back in. I'd merged master into this so that my `StringIndexed` changes would work with your new refactors on Table. I'll go back and manually compare master to this and fix it up. Probably what caused that eslint comment on 320 as well